### PR TITLE
Feature/analytics populate entities

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php [hunk 1]
```diff
@@ -80,7 +92,11 @@ class AnalyticsExport implements IsIdentifiable
     protected $providedEmailAddress = 0;
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: Would `boolean` be better suited?

Comment (brettminnie) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: These are for aggregate values, maybe a clearer name would be better

Basically this is to persist an analytic export in a given time window

Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: Yeah if possible a more descriptive name

Comment (brettminnie) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: Noted, I'll update in the next pr

Comment (eddiejaoude) [inline src/TransformCore/PHE/AnalyticsBundle/Entity/AnalyticsExport.php]: :+1:

**Pull request merged**